extensions/binding/org.eclipse.smarthome.binding.ntp/ESH-INF/i18n/ntp_de.properties

```diff
@@ -34,3 +34,6 @@ channel-type.config.ntp.string-channel.DateTimeFormat.description = Format f
 offline.conf-error-init-handler = Fehler bei der Initialisierung des NTP handler.
 offline.comm-error-unknown-host = Zeitserver {0} ist unbekannt. Systemzeit wird zur�ckgegeben.
 offline.comm-error-connection = Verbindung zum Zeitserver {0} kann nicht aufgebaut werden. Systemzeit wird zur�ckgegeben.
+
+# Discovery result
+#discovery.ntp.ntp.local.label = Local Time
```

Comment:
You can use 'Lokale Zeit' for geman translation. ;-)

Reply:
Done